test/performance/README.md

```diff
@@ -1,128 +1,188 @@
+### b) Collect the creation & update timestamps for workload benchmark (TO BE REVISED: WORK-IN-PROGRESS)
 
-To collect the creation and status update for a k8s object proceed as following:  
+To collect the creation and status update for the workload benchmark objects proceed as following:  
 
 1. Create a kubeconfig with contexts for the `wds1`, `its1` and `wec1`:
 
-For example: 
+   For example: 
 
-```bash
-$ kubectl config get-contexts
-CURRENT   NAME     CLUSTER                                          AUTHINFO                                                                 NAMESPACE
-          its1     its1-cluster                                     its1-admin                                                                default
-          wec1     <url>:port#                                      <defaul-value>                                                            default
-*         wds1     wds1-cluster                                     wds1-admin                                                                default
-```
+   ```bash
+   $ kubectl config get-contexts
+   CURRENT   NAME     CLUSTER                                          AUTHINFO                                                                 NAMESPACE
+            its1     its1-cluster                                     its1-admin                                                                default
+            wec1     <url>:port#                                      <defaul-value>                                                            default
+   *         wds1     wds1-cluster                                     wds1-admin                                                                default
+   ```
 
-Use the following command to rename the default context name for `wec1` workload execution cluster:
+   Use the following command to rename the default context name for `wec1` workload execution cluster:
 
-```bash 
-$ kubectl config rename-context <default-wec1-context-name> wec1
-```
+   ```bash 
+   $ kubectl config rename-context <default-wec1-context-name> wec1
+   ```
 
 2. Configure the following parameters in the `main` function in the `metrics_collectory.py` script
 
-```bash
-    kubeconfig = "kscore-config"  # path to the kubeconfig file
-    ns = "stress-test"  # name of the namespace where the workload are generated 
-    mypath="data/"  # path to the directory for the output files
-    run_ID="1" # run ID for your experiment 
-    freq="1"   # workload frequency parameter set in your experiment 
-    numPods=20 # total number of workload objects set in your experiment
-    method = "exp-RandomizedLoad" # clusterloader method used to generate your traffic
-    watch_interval = 3600  # watch for events that to occur within time interval threshold
-```
+   ```bash
+      kubeconfig = "kscore-config"  # path to the kubeconfig file
+      ns = "stress-test"  # name of the namespace where the workload are generated 
```

Comment:
Huh? I thought there is a configured number of namespaces used.

Reply:
Removing the context of that subsection for now as it will be completely revised in a NEW PR, as already stated on my comment above. Sorry for the confusion!